media-libs/ptex/ptex-2.1.28.ebuild

```diff
@@ -15,7 +15,7 @@ RDEPEND="sys-libs/zlib"
 DEPEND="${RDEPEND}
 	app-doc/doxygen"
 
-KEYWORDS="~amd64 ~x86"
+KEYWORDS="~amd64 ~ppc64 ~x86"
```

Comment:
Then should I remove ppc64 from everything that depends on it? Or if I mask the use flag for ppc64, it won't blow up?